src/main/scala/com/fortysevendeg/android/scaladays/ui/scheduledetail/ScheduleDetailActivity.scala

```diff
@@ -126,6 +131,7 @@ class ScheduleDetailActivity
   private def favoriteClick(eventTitle: String, name: String) = {
     favoriteChanged = true
     val isFavorite = preferenceServices.fetchBooleanPreference(PreferenceRequest[Boolean](name, false)).value
+    isFavoriteEvent = !isFavorite
```

Comment:
I was implementing the ticket and finally I changed the implementation and forget that. I remove it now. Thanks!